idaes/models_extra/column_models/MEAsolvent_column.py

```diff
@@ -0,0 +1,1483 @@
+
+        init_log.info('Step 4b: Isothermal chemical absoption')
+        init_log.info_high("Adding mass transfer to material balances")
+
+        blk.vapor_phase.mass_transfer_term.unfix()
+        blk.liquid_phase.mass_transfer_term.unfix()
+        blk.vapor_mass_transfer_eqn.activate()
+        blk.liquid_mass_transfer_eqn.activate()
+
+        # Fix this
+        with idaeslog.solver_log(solve_log, idaeslog.DEBUG) as slc:
+            res = opt.solve(blk, tee=slc.tee)
+        init_log.info_high(
+            "Step 4 complete: {}.".format(idaeslog.condition(res)))
+
+        # ---------------------------------------------------------------------
+        init_log.info('Step 5: Adiabatic chemical absoption')
+        init_log.info_high("Isothermal to Adiabatic ")
+
+        # Unfix heat transfer terms
+        blk.heat_transfer_coeff.unfix()
+        for c in ["heat_transfer_Ackmann_correction"]:
+            getattr(blk, c).activate()
+            
+        for k in blk.heat_transfer_Ackmann_correction:
+            calculate_variable_from_constraint(
+                    blk.heat_transfer_coeff[k],
+                    blk.heat_transfer_Ackmann_correction[k])
+        
+        with idaeslog.solver_log(solve_log, idaeslog.DEBUG) as slc:
+            res = opt.solve(blk, tee=slc.tee)
+            
+        blk.vapor_phase.heat.unfix()
+        blk.liquid_phase.heat.unfix()
+        blk.vapor_phase.enthalpy_transfer.unfix()
+        blk.liquid_phase.enthalpy_transfer.unfix()
+
+        # Activate heat transfer equations
+        for c in [
+                  "heat_transfer_eqn1",
+                  "heat_transfer_eqn2",
+                  "enthalpy_transfer_eqn1",
+                  "enthalpy_transfer_eqn2"]:
+            getattr(blk, c).activate()
+
+        with idaeslog.solver_log(solve_log, idaeslog.DEBUG) as slc:
+            res = opt.solve(blk, tee=slc.tee)
+
+        init_log.info_high(
+            "Step 5 complete: {}.".format(idaeslog.condition(res)))
+
+        # ---------------------------------------------------------------------
+        blk.vapor_phase.release_state(flags=vflag)
+        blk.liquid_phase.release_state(flags=lflag)
+
+        if (res.solver.termination_condition != TerminationCondition.optimal or
+                res.solver.status != SolverStatus.ok):
+            raise InitializationError(
+                f"{blk.name} failed to initialize successfully. Please check "
+                f"the output logs for more information.")
+            
+        blk.calculate_scaling_factors_unit_model()
+        iscale.calculate_scaling_factors(blk)
+        
+        # ---------------------------------------------------------------------
+        init_log.info('Step 6: Velocity constraints')
+        blk.velocity_vap.unfix()
+        blk.velocity_liq.unfix()
+        
+        for c in velocity_constraints:
+            getattr(blk, c).activate()
+            
+        use_idaes_solver_configuration_defaults()
```

Comment:
I do not think you should be changing solver options in the middle of a solve like this (unless it is absolutely necessary). Could you have just set these from the start?